src/common/metrics/case_density.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { COLOR_MAP } from 'common/colors';
+import { Level, LevelInfoMap } from 'common/level';
+import { Projection } from 'common/models/Projection';
+import { getLevel, Metric } from 'common/metric';
+import { formatDecimal, formatPercent, formatInteger } from 'common/utils';
+import ExternalLink from 'components/ExternalLink';
+
+export const METRIC_NAME = 'Daily new cases per 100k population';
+
+export const CASE_DENSITY_LEVEL_INFO_MAP: LevelInfoMap = {
+  [Level.LOW]: {
+    level: Level.LOW,
+    upperLimit: 1,
+    name: 'Low',
+    color: COLOR_MAP.GREEN.BASE,
+    detail: () => 'COVID is being effectively contained',
+  },
+  [Level.MEDIUM]: {
+    level: Level.MEDIUM,
+    upperLimit: 10,
+    name: 'Medium',
+    color: COLOR_MAP.ORANGE.BASE,
+    detail: () => 'COVID not contained, but at low levels',
+  },
+  [Level.HIGH]: {
+    level: Level.HIGH,
+    upperLimit: 25,
+    name: 'High',
+    color: COLOR_MAP.ORANGE_DARK.BASE,
+    detail: () => 'Very large number of new cases',
+  },
+  [Level.CRITICAL]: {
+    level: Level.CRITICAL,
+    upperLimit: Infinity,
+    name: 'Critical',
+    color: COLOR_MAP.RED.BASE,
+    detail: () => 'Dangerous number of new cases',
+  },
+  [Level.UNKNOWN]: {
+    level: Level.UNKNOWN,
+    upperLimit: 0,
+    name: 'Unknown',
+    color: COLOR_MAP.GRAY.BASE,
+    detail: () => 'Insufficient data to assess',
+  },
+};
+
+export const CASE_DENSITY_DISCLAIMER = '';
+
+export function renderStatusText(projection: Projection): React.ReactElement {
+  const {
+    locationName,
+    currentCaseDensity,
+    currentDailyDeaths,
+    totalPopulation,
+  } = projection;
+  if (
+    currentCaseDensity === null ||
+    currentDailyDeaths === null ||
+    totalPopulation === null
+  ) {
+    return (
+      <React.Fragment>
+        {`Not enough case data is available to generate ${METRIC_NAME.toLowerCase()}.`}
```

Comment:
Are you intentionally using JavaScript string interpolation instead of JSX expression syntax?

This could just be:

```
<React.Fragment>
Not enough case data is available to generate {METRIC_NAME.toLowerCase()}.
</React.Fragment>
```

right?

Same for all the other fragments you added...  If you're just trying to minimize churn for now, that's fine. It just looks a bit odd.

Reply:
😅 I think I momentarily forgot that this syntax exists, updating!